_episodes/005-rust-1.38.0.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "Rust in Production: An Interview with Armin Ronacher"
+date: 2019-09-17T18:00:00+0000
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e005-rust-1.38.0.mp3
+duration: "34:03" #audio length
+length: "24517728" #filesize in byte
+
+# https://github.com/jekyll/jekyll/issues/7744
+layout: episode
+---
+
+Jon and Ben review the changes introduced by [the Rust 1.38 release](https://blog.rust-lang.org/2019/09/26/Rust-1.38.0.html).
+
+Get in touch with us if you'd like to be interviewed, propose a topic for an episode, or help out!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+* [@1:15] - Pipelined compilation
+
```

Comment:
I don't know if these spaces between the lines should be there? I think markdown requires consecutive lines to make them a list?

Reply:
Hm, they look fine in the rendered output on Github?

Reply:
It's not actually rendered as _one_ list, it's rendered as multiple single-item lists. This probably doesn't matter a tonne, but it's not _quite_ right.